README.md

```diff
@@ -113,14 +114,33 @@ mm.send("lights", true);   // Turn on the lights
 
 <div id="mmanager_on"><h4>MessageManager.on(<i>messageName, callback</i>)</h4></div>
 
-Sets a message listener function (*callback*) for the specified *messageName*. The callback function takes two parameters: *message* (the message) and *reply* (a function that can be called to reply to the message).
+Sets a message listener function (*callback*) for the specified *messageName* or for unhandled messages if *messageName* is null.
```

Comment:
```suggestion
If the `messageName` argument is not `null`, sets the name-specific message handler. Otherwise sets the generic message handler. The generic message handler captures any messages not handled by any of the name-specific callbacks.
```